cog/version_info.py

```
@@ -0,0 +1,131 @@
+        return discord.Embed(
+            color=0xFF24CF,
+            title="版本資訊",
+            description=description,
+            fields=self._embed_fields_installed_packages(),
+            author=discord.EmbedAuthor(
+                name="中電喵",
+                icon_url=self.bot.user.display_avatar.url,
+            ),
+            thumbnail=self._SCAICT_UWU_IMAGE,
+            footer=discord.EmbedFooter(
+                text=datetime.datetime.now(tz=datetime.timezone.utc).strftime(
```

Ruff (0.14.10)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)